migrate2rocky/migrate2rocky.sh

```diff
@@ -212,8 +221,8 @@ bin_check() {
 
     local -a missing bins
     bins=(
-	rpm dnf awk column tee tput mkdir cat arch sort uniq
-	rmdir rm head curl sha512sum mktemp systemd-detect-virt
+	rpm dnf awk column tee tput mkdir cat arch sort uniq rmdir
```

Comment:
tab instead of blanks

Reply:
You don't need to repeat this for every commit that's made.  There will be another commit with formatting/spelling changes coming up.